src/vm/types/ecdsa_instance_def.zig

```diff
@@ -28,4 +34,54 @@ pub const EcdsaInstanceDef = struct {
             .n_hash_bits = 251,
         };
     }
+
+    pub fn _cells_per_builtin(_: Self) u32 {
+        return CELLS_PER_SIGNATURE;
+    }
+
+    pub fn _range_check_units_per_builtin(_: *const Self) u32 {
```

Comment:
Following Ziglang naming convention can we rename this one like: `rangeCheckUnitsPerBuiltin`?

Reply:
Updated